modules/server/src/network/gql.ts

```diff
@@ -1,13 +1,26 @@
 import axios from 'axios';
-import { print } from 'graphql';
+import { ASTNode, print } from 'graphql';
 
-export const createGqlQuery = ({ query: queryAST, variables }) => {
+/**
+ * Creates a graphql query string with variables for use in a POST request
+ * @returns string
+ */
+export const createGqlQuery = ({
+	query: queryAST,
+	variables,
+}: {
+	query: ASTNode;
+	/**
+	 * TODO: Add typing for variables. Type will be dependant on gql endpoint being queried.
+	 */
+	variables: unknown;
+}) => {
 	// queries need to be strings when using regular http
 	const query = print(queryAST);
 	return JSON.stringify({ query, variables });
 };
 
-export const fetchGql = ({ url, gqlRequest }) => {
+export const fetchGql = ({ url, gqlRequest }: { url: string; gqlRequest: { query: string } }) => {
```

Comment:
TSDoc will be wanted before feature branch is complete.

Reply:
I think your TSDoc is incorrect, instead of returning an axios instance that you can use to send future requests, this sends your defined request with axios and returns the AxiosResponse (or throws error, stupid axios).

Reply:
Great feedback, useful insight... but let's try keeping unhelpful personal opinions to a minimum.